tests/unit/package_managers/bundler/test_main.py

```
@@ -305,3 +306,46 @@ def test_download_gems_with_proxy_credentials_adds_auth_header(
         concurrency_limit=5,
         headers={expected_url: {"Authorization": aiohttp.encode_basic_auth("user", "password")}},
     )
+
+
+@pytest.mark.parametrize(
+    "proxy_url, gem_deps, should_log_proxy",
```

Ruff (0.16.3)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)